### PR TITLE
LPS-75955 Baseline

### PR DESCRIPTION
**Pull request by 4lejandrito**
/cc @sergiogonzalez

### CODE REVIEW

File: modules/apps/forms-and-workflow/polls/polls-service/src/main/java/com/liferay/polls/internal/verify/PollsServiceVerifyProcess.java [hunk 1]
```diff
@@ -54,6 +56,13 @@ protected void verifyAuditedModels() throws Exception {
 		}
 	}
 
+	protected void verifyGroupedModels() throws Exception {
```
Comment (4lejandrito) [inline modules/apps/forms-and-workflow/polls/polls-service/src/main/java/com/liferay/polls/internal/verify/PollsServiceVerifyProcess.java]: Hi @leoadb, can you **verify** if this changes are ok in `polls-service`? (Or point us to who is in charge of it?

Thanks!

**Comment by 4lejandrito** [comment]
ci:test

File: modules/apps/collaboration/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/service/MBDiscussionMBMessageServiceWrapper.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+				return _userLocalService.getDefaultUser(
+					CompanyThreadLocal.getCompanyId());
+			}
+			catch (Exception e) {
+				throw pe;
+			}
+		}
+	}
+
+	private PermissionChecker _getPermissionChecker()
```
Comment (4lejandrito) [inline modules/apps/collaboration/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/service/MBDiscussionMBMessageServiceWrapper.java]: Hi @brianchandotcom,

These private methods will go away as soon as we extract the `MBMessage` service. We will include these changes in the new service and use the inherited version of said methods.

**Comment by 4lejandrito** [comment]
Hi @brianchandotcom,

This has been opened for a few days. We didn't get a response in one of the comments but we are happy with it and would like this to move forward.

Thansk!

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/6b6357f89b...e4dc5335b6

**Comment by brianchandotcom** [comment]
Thx @4lejandrito just one minor change in upstream. Thx.

**Pull request closed without merging**